src/promptflow/promptflow/azure/_constants/_flow.py

```diff
@@ -20,7 +20,7 @@ class FlowJobType:
 
 CLOUD_RUNS_PAGE_SIZE = 25  # align with UX
 
-SESSION_CREATION_TIMEOUT_SECONDS = 10 * 60  # 10 minutes
+SESSION_CREATION_TIMEOUT_SECONDS = 20 * 60  # 20 minutes
```

Comment:
Lisa comment that we should not change this, as we control the user experience, and should not change this as this not improves the user experience.  @D-W-  @eniac871 

However we should make this configurable for Runner,

Reply:
Added env var SESSION_CREATION_TIMEOUT_SECONDS to allow runner configure